movies/src/main/java/com/caueferreira/movies/data/MovieRaw.kt

```diff
@@ -0,0 +1,25 @@
+package com.caueferreira.movies.data
+
+import java.util.*
+
+data class MovieRaw(
+    val id: Int, val adult: Boolean, val backdropPath: String?, val budget: Int, val homepage: String?,
```

Comment:
Break line for each attribute as per kotlin style-design

Reply:
Sure